map/src/menu/tracks/TracksMenu.jsx

```diff
@@ -162,17 +252,98 @@ export default function TracksMenu() {
+                <Toolbar className={styles.toolbar}>
+                    <IconButton variant="contained" type="button" className={styles.icon} onClick={closeTrackMenu}>
+                        <CloseIcon />
+                    </IconButton>
+                    <Typography component="div" className={styles.title}>
+                        {DEFAULT_GROUP_NAME}
+                    </Typography>
+                    <Tooltip key={'sort_tracks'} title="Sort tracks" arrow placement="bottom-end">
+                        <IconButton
+                            variant="contained"
+                            type="button"
+                            className={styles.icon}
+                            onClick={() => setOpenSort(true)}
+                            ref={anchorEl}
+                        >
+                            <TimeIcon />
+                        </IconButton>
+                    </Tooltip>
+                    <Tooltip key={'import_track'} title="Import track" arrow placement="bottom-end">
+                        <span>
+                            <LocalGpxUploader>
+                                <IconButton component="span" variant="contained" type="button" className={styles.icon}>
+                                    <ImportIcon />
+                                </IconButton>
+                            </LocalGpxUploader>
+                        </span>
+                    </Tooltip>
+                </Toolbar>
+            </AppBar>
+            {ctx.gpxLoading ? (
+                <Loading />
+            ) : (
+                <>
+                    {ctx.tracksGroups?.length > 0 || defaultGroup?.length > 0 ? (
+                        <Box
+                            minWidth={ctx.infoBlockWidth}
+                            maxWidth={ctx.infoBlockWidth}
+                            sx={{
+                                overflowX: 'hidden !important',
+                                overflowY: 'auto !important',
+                                maxHeight: `${height - 120}px`,
+                            }}
+                        >
+                            {visibleTracksOpen() && (
+                                <VisibleGroup visibleTracks={visibleTracks} setVisibleTracks={setVisibleTracks} />
+                            )}
+                            {ctx.tracksGroups &&
+                                (sortGroups.length > 0 ? sortGroups : ctx.tracksGroups).map((group, index) => {
+                                    return <CloudTrackGroup key={group.name + index} index={index} group={group} />;
+                                })}
+                            {defaultGroupItems}
+                        </Box>
+                    ) : (
+                        <Empty
+                            title={'You don’t have track files'}
+                            text={'You can import, create or record track files using OsmAnd App.'}
```

Comment:
Just text proposal:

You can sync your tracks with OsmAnd Cloud or use the Create/Import buttons.